guides/micronaut-security-jwt-cookie/groovy/src/test/groovy/example/micronaut/AuthenticationSpec.groovy

```diff
@@ -1,73 +0,0 @@
-package example.micronaut
-
-import geb.spock.GebSpec
-import io.micronaut.runtime.server.EmbeddedServer
-import io.micronaut.test.extensions.spock.annotation.MicronautTest
-
-import jakarta.inject.Inject
-
-//tag::clazz[]
-@MicronautTest // <1>
-class AuthenticationSpec extends GebSpec {
```

Comment:
did we remove this file because it was not being executed?

Reply:
no, all 3 languages use the same Geb tests so I moved them to shared src/test in https://github.com/micronaut-projects/micronaut-guides/pull/479/commits/52690cd19f1d273ce735861243b6046dcd34d122